### ISSUE TITLE
Manual localization information for old pymms and pyFR data

### ISSUE THREAD

**Issue by HonghuiZ** [pennmem/cmlreaders#198]
For some patients, there's localization file named hipp_only.txt, which donates the channels were manually localized in the hippocampus. The file has a few lines and each line matches with the channel number of a electrode in the hippocampus. The file is always in dataX/eeg/subjectName/tal/hipp_only.txt, but not all subjects have that information (file) there. The following is one example. Wouldn't it be nice to add the to the location tag? 

/rhino/data4/eeg/TJ020/tal/hipp_only.txt
74
75
76
79
85
86

**Comment by mivade** [comment]
I guess I'm still a little unsure of what this is. Is it just indicating a boolean value that a contact is in the hippocampus?

**Comment by HonghuiZ** [comment]
Yes. The numbers appear in the file indicating channels inside the hippocampus, and those don't are not in the hippocampus.  It could be transformed to a boolean values matching to every electrode (channel).
